src/sys.c

```diff
@@ -742,6 +742,30 @@ DLLEXPORT jl_sym_t* jl_get_ARCH()
     return ARCH;
 }
 
+DLLEXPORT size_t jl_maxrss()
```

Comment:
I had that in the first commit, but Windows is not supported in the libuv interface - it returns 0 for maxrss size. Hence decided to just have a simple solution that works on all 3 test platforms.